gitsync/sync/fetch_pipeline.py

```diff
@@ -39,6 +39,37 @@ def on_update(self, data: enc.BinaryStr, respond_to: typ.Optional[bytes]):
         if not self.in_process:
             self.in_process = True
             aio.create_task(self.after_update(ref_updates, respond_to))
+    
+    async def after_bmeta_commit(self, ref_updates: typ.Dict[str, bytes], force: bool=False):
```

Comment:
How can you force a push to bmeta? Who is authorized to push and how can you maintain the consistency after a force push?
In our design:
1. the **only** branch that can go backwards is code branch `refs/head/*` because it is managed by `bmeta`.
2. mergable branch can automatically merge multiple compatible commits.
3. all other branches **must reject** any commit that does not contain local head.
Recall that the reason we have `bmeta` is exactly to handle the force push of `head`.